XLS-0033d-CFTs/README.md

```diff
@@ -514,35 +515,72 @@ We propose using the following format for CFT amounts::
 
 ```json
 "amount": {
-  "issuer": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
-  "cft_asset": "USD",
+  "cft_issuance_id": "00070C4495F14B0E44F78A264E41713C64B5F89242540EE255534400000000000000",
   "value": "1"
 }
 ```
 
-The idea behind this format is that it adds only one new subfield to the `amount` field, but still distinguishes between itself and Issued Currency amounts. Using the CFT ID directly would not allow us to easily find the underlying `CFTokenIssuance` object because we would still need the `issuer` to know where to look for it.
+Since the `AssetCode` is an optional field, the `CFTokenIssuanceID` will be used to uniquely identify the CFT during a Payment transaction.
+
+### 1.3.5 The **`CFTokenTrust`** Transaction
+This transaction allows a holder to own a `CFToken`, creating a `CFToken` object for the holder with zero balance. 
+
+If the issuer has set `lsfRequiresAuthorization`(allowlisting) on the `CFTokenIssuance`, then the issuer must submit a `CFTokenTrust` transaction as well in order to give permission to the holder. If `lsfRequiresAuthorization` is not set and the issuer attempts to submit this transaction, it will fail. Read more about allowlisting in Section 1.8.0. 
+#### 1.3.5.1 CFTokenTrust
+| Field Name      | Required?          | JSON Type | Internal Type |
+| --------------- | ------------------ | --------- | ------------- |
+| `Account`       | :heavy_check_mark: | `string`  | `ACCOUNTID`   | 
+
+This address can indicate either an issuer or a potential holder of a CFT.
+
+| Field Name         | Required? | JSON Type | Internal Type |
+| ------------------ | --------- | --------- |---------------|
+| `TransactionType`  | ️ ✔        | `object`  | `UINT16`      |
+
+Indicates the new transaction type **`CFTokenTrust`**. The integer value is `29 (TODO)`.
+
+| Field Name  | Required? | JSON Type | Internal Type |
+| ----------- | --------- | --------- | ------------- |
+| `CFTokenIssuanceID` |  ✔️  | `string`  | `UINT256`     | 
+
+Indicates the ID of the CFT involved. 
+
+| Field Name  | Required? | JSON Type | Internal Type |
+| ----------- | --------- | --------- | ------------- |
+| `CFTokenHolder` |    | `string`  | `ACCOUNTID`     | 
 
-### 1.3.5 The **`AccountDelete`** Transaction
+This field is only used for authorization/allowlisting. It's an optional field that specifies the holder's address whom the issuer wants to authorize.
+
+| Field Name      | Required?          | JSON Type | Internal Type |
+| --------------- | ------------------ | --------- | ------------- |
+| `Flag`          | :heavy_check_mark: | `string`  | `UINT64`      | 
+
+ 
+#### 1.3.3.5.12 CFTokenTrust Flags
```

Comment:
If the holder calls `CFTokenTrust` with the `tfUntrust` flag, it will delete holder's CFToken as long as the amount is zero. Are you talking about introducing a new `CFTokenDelete` transaction? Because we currently don't have transaction that's called this.

Reply:
I'm not sure - I'll revisit this idea later once this PR is merged.